ArduPlane/GCS_Mavlink.cpp

```diff
@@ -157,6 +157,36 @@ void GCS_MAVLINK_Plane::send_attitude() const
         omega.z);
 }
 
+void GCS_MAVLINK_Plane::send_attitude_target() 
+{
+#if HAL_QUADPLANE_ENABLED
```

Comment:
This should go around the outside of the method so it doesn't cost us bytes when quadplane isn't available.